### DISCUSSION TITLE
Normal hello world javascript program exists with -1 code

### DISCUSSION THREAD

**Discussion by andreea-7** [nodejs/help#4780] [Q&A]
### Version

v18.18.0

### Platform

Microsoft Windows NT 10.0.19045.0 x64

### What steps will reproduce the bug?

The error doesn't really belong to the code itself.  I think it's an error related to the node.exe installed file.
My program consists of a single line of code, namely this one: console.log('Hello World'); 
After running it, the console shows up, it correctly displays "Hello World", but it exists with code -1. Just like that.
`The program 'app.js' has exited with code -1 (0xffffffff).`
This is displayed in the output downward window.

### How often does it reproduce? Is there a required condition?

It's been happening since I installed node.js on my machine.

### What is the expected behavior? Why is that the expected behavior?

It should exit with code 0 because I installed node.exe and I linked it with visual studio.

### What do you see instead?

I see that it exits with code -1...

**Comment by bnoordhuis** [comment]
Converted to a discussion because this is a local problem, not a node bug. What happens when you open a shell and type `node.exe app.js`?

**Comment by andreea-7** [comment]
throw err;
  ^

Error: Cannot find module 'C:\Users\Asus\app.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1077:15)
    at Module._load (node:internal/modules/cjs/loader:922:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:86:12)
    at node:internal/main/run_main_module:23:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v18.18.0




This is what it displays...

**Reply by bnoordhuis** [reply]
Apply some critical thinking, okay? :-)

That error tells you that...? Hint: make sure you're in the right directory or use the full path.
